src/System.Buffers.Experimental/System/Buffers/Sequences/ReadOnlyBytes.cs

```diff
@@ -1,383 +1,384 @@
-                item = node.Memory.Slice(index);
-                if (ReferenceEquals(node, _end))
-                {
-                    item = item.Slice(0, _endIndex - index);
-                    if (advance) position = default;
-                }
-                else
-                {
-                    if (advance) position = new Position(node.Next, 0);
-                }
-                return true;
-            }
-
-            var om = _start as OwnedMemory<byte>;
-            if (om != null)
-            {
-                var start = (int)position;
-                var length = _endIndex - (int)position;
-                item = om.Memory.Slice(start, length);
-                if (advance) position = default;
-                return true;
-            }
-
-            throw new NotImplementedException();
-        }
-
-        public Position Seek(Position origin, long offset)
-        {
-            if (offset < 0) throw new ArgumentOutOfRangeException(nameof(offset));
-
-            switch (Kind)
-            {
-                case Type.Array:
-                    {
-                        var (array, index) = origin.Get<byte[]>();
-                        if (index + offset > array.Length) throw new ArgumentOutOfRangeException(nameof(offset));
-                        return new Position(array, (int)(index + offset));
-                    }
-                case Type.OwnedMemory:
-                    {
-                        var (om, index) = origin.Get<OwnedMemory<byte>>();
-                        if (index + offset > om.Length) throw new ArgumentOutOfRangeException(nameof(offset));
-                        return new Position(om, (int)(index + offset));
-                    }
-                case Type.MemoryList:
-                    var previous = origin;
-                    while (TryGet(ref origin, out var memory))
-                    {
-                        var length = memory.Length;
-                        if (length < offset)
-                        {
-                            offset -= length;
-                            previous = origin;
-                        }
-                        else
-                        {
-                            var (segment, index) = previous.Get<IBufferList>();
-                            return new Position(segment, (int)(index + offset));
-                        }
-                    }
-                    throw new ArgumentOutOfRangeException(nameof(offset));
-                default:
-                    throw new NotSupportedException();
-            }
-        }
-
-        enum Type : byte
-        {
-            Array,
-            OwnedMemory,
-            MemoryList,
-        }
-
-        public override string ToString()
-        {
-            var buffer = ToSpan().ToArray();
-            return Encoding.UTF8.GetString(buffer, 0, buffer.Length);
-
-        }
-    }
-}
+﻿//// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why is this commented out?

Reply:
@davidfowl, Sarah Faraphallower said it wasn't required...

Reply:
Won't immediately say I disagree either, my 3 Sat and CRT/JMT logic tables work without it....

Reply:
@davidfowl, something went very wrong with the merge. I really don't understand how the tests passed. Anyway, I meant to delete this whole file, and I will shortly.